Day-I-Social-Card/social-card/src/components/CardComponents/CardBanner.js

```diff
@@ -4,7 +4,7 @@ import './Card.css';
 const CardBanner = (props) => {
     return (
         <div>
-        <img src={'https://ibin.co/3wnC6SgIOJud.png'} alt="banner" />
+        <img className = 'banner' src={'https://ibin.co/3wnC6SgIOJud.png'} alt="banner" />
```

Comment:
You can omit the `{}` around your img URL. If you stored that img URL as a prop, then you would _have_ to use JSX {} to get that evaluated.